docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -71,19 +71,11 @@ Returns the short string `'VALID'` if valid, otherwise it reverts.
 ```javascript
 use openzeppelin::account::Account;
 ```
-Account contract implementation extending xref:ISRC6[`ISRC6`].
+Account component extending xref:ISRC6[`ISRC6`].
 
 [.contract-index]
-.Constructor
+.Embeddable Functions
 --
-* xref:#Account-constructor[`++constructor(self, _public_key)++`]
---
-
-[.contract-index]
-.External Functions
---
-* xref:#Account-\\__validate_deploy__[`++__validate_deploy__(self, hash, signature)++`]
-
 .SRC6Impl
 
 * xref:#Account-\\__execute__[`++__execute__(self, calls)++`]
```

Comment:
does the component really include this impl? in our examples we're importing it independently

Reply:
Which implementation? If you are referring to `DeployableImpl` it does include it.